Source/gg2/Objects/Overlays/Gamemodes/ControlPointHUD.events/Create.xml

```diff
@@ -21,8 +21,10 @@ global.totalControlPoints = instance_number(ControlPoint);
 
 if instance_exists(ControlPointSetupGate)
 {
-    global.setupTimer=1800;
+    global.setupTimer=900;
     mode=1;
+    // 3 minutes
+    timeLimit = 3*30*60;
```

Comment:
Besides, `timer` is set to `timeLimit` above this line, so will we actually start with the correct `timer` value?